src/test/java/com/learning/selenium/app/Application.java

```diff
@@ -0,0 +1,99 @@
+package com.learning.selenium.app;
+
+import com.learning.selenium.pages.google.GoogleCloudHeader;
+import com.learning.selenium.pages.google.GoogleCloudMainPage;
+import com.learning.selenium.pages.google.GoogleCloudPricingCalcPage;
+import com.learning.selenium.pages.google.GoogleCloudSearchResultsPage;
+import com.learning.selenium.pages.pastebin.PasteMainPage;
+import com.learning.selenium.pages.yandex.YandexDiskPage;
+import com.learning.selenium.pages.yandex.YandexLoginPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.BrowserType;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class Application {
+
+    public WebDriver driver;
+    public Actions actions;
+    private PasteMainPage pasteMainPage;
+    private GoogleCloudMainPage googleCloudMainPage;
+    private GoogleCloudHeader googleCloudHeader;
+    private GoogleCloudSearchResultsPage googleCloudSearchResultsPage;
+    private GoogleCloudPricingCalcPage googleCloudPricingCalcPage;
+    private YandexLoginPage yandexLoginPage;
+    private YandexDiskPage yandexDiskPage;
+
+    public Application(String browser, String seleniumServer) {
+        if ("".equals(seleniumServer)) {
```

Comment:
This logic might be put on another layer as I explained before

Reply:
Updated